cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -143,3 +143,6 @@ spec:
 #        storeType: filestore
 #    - name: "172.17.4.301"
 #      deviceFilter: "^sd."
+  disruptionManagement: #The section for configuring management of daemon disruptions
+    managePodBudgets: false #if `true`, the operator will create and manage PodDsruptionBudgets for OSD, Mon, RGW, and MDS daemons. OSD PDBs are managed dynamically via the strategy outlined in the [design](https://github.com/rook/rook/blob/master/design/ceph-managed-disruptionbudgets.md). The operator will block eviction of OSDs by default and unblock them safely when drains are detected.
+    #osdMaintenanceTimeout: is a duration in minutes that determines how long an entire failureDomain like `region/zone/host` will be held in `noout` (in addition to the default DOWN/OUT interval) when it is draining. This is only relevant when  `managePodBudgets` is `true`. The default value is `30` minutes.
```

Comment:
How about uncommenting this and giving it a default value? Since the default for `managePodBudgets` is false, this won't take effect anyway.